samples/connectors/src/main/java/quarks/samples/connectors/iotf/IotfSensors.java

```diff
@@ -119,20 +119,8 @@ public static void simulatedSensors(IotDevice device, boolean print) {
      * @param print true to print generated heartbeat tuples to System.out.
      */
     public static void heartBeat(IotDevice device, boolean print) {
-        // In addition create a heart beat event to
-        // ensure there is some immediate output and
-        // the connection to IoTF happens as soon as possible.
-        TStream<Date> hb = device.topology().poll(() -> new Date(), 1, TimeUnit.MINUTES);
-        // Convert to JSON
-        TStream<JsonObject> hbj = hb.map(d -> {
-            JsonObject j = new  JsonObject();
-            j.addProperty("when", d.toString());
-            j.addProperty("hearbeat", d.getTime());
-            return j;
-        });
-        if (print)
-            hbj.print();
-        device.events(hbj, "heartbeat", QoS.FIRE_AND_FORGET);
+      HeartBeat.addHeartBeat(device, 1, TimeUnit.MINUTES,
+          "heartbeat", stream -> { if (print) stream.print(); });
```

Comment:
Found the use of 'stream' a little confusing, since it's the tuple on the stream, not the stream itself.